### PR TITLE
Issue 3 create user module & entity

### PR DESCRIPTION
**Pull request by mostafa8026**

### CODE REVIEW

**Review by miladworkac — APPROVED** [review]

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Databse configuration
+DB_HOST="localhost"
+DB_TYPE=mssql
+DB_PORT=1433
+DB_USER_NAME=user1
+DB_PASSWORD='!!!!!!!'
+DB_DATABASE=diffal
```
Comment (fahimi85) [inline .env.sample:7]: A minus red sign is seen. Maybe it is better to press Enter after the last line of code. @mostafa8026

**Review by fahimi85 — APPROVED** [review]
very excellent

**Review by alieidelkhani — APPROVED** [review]

**Review by Rzix — COMMENTED** [review]
perfect╰(*°▽°*)╯

File: src/user/user.service.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  }
+
+  update(id: number, updateUserDto: UpdateUserDto) {
+    return `This action updates a #${id} user`;
+  }
+
+  remove(id: number) {
+    return `This action removes a #${id} user`;
+  }
+}
```
Comment (Rzix) [inline src/user/user.service.ts:29]: 👍

**Pull request merged**